addons/point_of_sale/static/src/app/printer/render_service.js

```diff
@@ -90,6 +90,20 @@ const applyWhenMounted = async ({ el, container, callback }) => {
     return res;
 };
 
+const sanitizeNodeText = (node) => {
+    const traverse = (element) => {
+        if (element.nodeType === Node.TEXT_NODE) {
+            element.textContent = element.textContent.replace(/[\x00-\x08\x0B\x0C\x0E-\x1F]/g, "");
+            return;
+        }
+        for (let child of element.childNodes) {
+            traverse(child);
+        }
+    }
+    traverse(node);
+    return node;
+}
```

Comment:
Do you need the inner function ?
```suggestion
const sanitizeNodeText = (element) => {
        if (element.nodeType === Node.TEXT_NODE) {
            element.textContent = element.textContent.replace(/[\x00-\x08\x0B\x0C\x0E-\x1F]/g, "");
            return;
        }
        for (let child of element.childNodes) {
            sanitizeNodeText(child);
        }
    }
}
```

Reply:
@vlst-odoo nope I don't, fixed it ;)